CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ChangeLog
 =========
 
+5.1.2 (2020-10-03)
+------------------
+
+* #82: Added support for PHP 8.0 (@phil-davis)
```

Comment:
Actually 5.1.1 declared the PHP 8.0 "experimental" support.
This adds the testing...
But probably worth doing a release here just so that the latest `composer.json` ... is released.